src/content/1.7/faq/stock.md

```diff
@@ -0,0 +1,77 @@
+| `reserved_quantity` | This indicates how many of your physical quantity is currently reserved for your pending orders, the product has been bought by a customer but it is still in your inventory because the order is not paid yet, or it is paid but the package has not been sent yet. |
+| `out_of_stock`      | This is the configuration defining how you want to handle this product in the FO when it is out of stock (available stock is too low). {{< code php >}}use PrestaShop\PrestaShop\Core\Domain\Product\Stock\ValueObject\OutOfStockType;
+
+OutOfStockType::OUT_OF_STOCK_NOT_AVAILABLE = 0; // Not available when out of stock
+OutOfStockType::OUT_OF_STOCK_AVAILABLE = 1; // Available when out of stock
+OutOfStockType::OUT_OF_STOCK_DEFAULT = 2; // Use default configuration
+{{< /code >}}|
+| `location`          | The location where the product is stored, to help merchant handle their storage and prepare their packages. |
+| `depends_on_stock`  | **Deprecated** It was used for advanced stock management which is no longer a feature. |
+
+{{% notice note %}}
+You will probably notice that usually ```physical_quantity = available_quantity + reserved_quantity;```
+{{% /notice %}}
+
+**Q:** When is the StockAvailable object/table updated?
+
+**A:** That's a broad question but basically for any action that should logically have an impact on the stock. When a product quantity is modified in BO, when an order is created, when the order is delivered, ... However here is a little chronology of stock quantities during a full checkout process:
+
+| Scenario Step | quantity | physical_quantity | reserved_quantity |   |
+| ------------- | -------- | ----------------- | ----------------- | - |
+| `Great Product` is created in the BO with a quantity of 50 | `50` | `50` | `0` | |
+| A user decides to add 3 `Great Product` into his cart      | `50` | `50` | `0` | For now the product is only in a cart, nothing is reserved anyone can still buy one of the 50 remaining products. |
+| The user goes through the whole checkout process and chooses to pay by check. | `50` | `47` | `3` | The order has been created, but the payment is not accepted yet. |
+| A few days later, the check is received, and the merchant changes the Order status to `Payment accepted` | `50` | `47` | `3` | The product is now paid, but it still has not left your shelves. |
+| The package is prepared and delivered to a Carrier, the order status is switched to `Delivered` or `Shipped` | `47` | `47` | `0` | The product is no longer reserved, it has actually been sent to the customer. So reserved and physical quantities can be decreased. |
+
+**Q:** So where can I see and modify my Stock?
+
+**A:** Although you can change the quantity directly in your product page we strongly recommend using the `Catalog > Stock` section in your BO because it is based on `StockMovement` which is more stable than direct edition.
```

Comment:
```suggestion
**A:** Although you can change the quantity directly on your product page, we strongly recommend using the `Catalog > Stock` section in your BO because it is based on `StockMovement` which is safer than a direct edition (read below).
```